From 5ffdd4424a9bbdf89b213601eaa5c386cdef08ad Mon Sep 17 00:00:00 2001
Message-ID: <5ffdd4424a9bbdf89b213601eaa5c386cdef08ad.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:21:04 +0100
Subject: [PATCH 07/19] gnu: go-github-com-golangplus-testing: Move to (gnu
 packages golang-check).

* gnu/packages/golang.scm (go-github-com-golangplus-testing): Move from
here...
* gnu/packages/golang-check.scm: ... to here.
---
 gnu/packages/golang-check.scm | 24 ++++++++++++++++++++++++
 gnu/packages/golang.scm       | 22 ----------------------
 2 files changed, 24 insertions(+), 22 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 8c6bf03f74..414423a5ef 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -2,6 +2,7 @@
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2019 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2020 Jakub Kądziołka <kuba@kadziolka.net>
+;;; Copyright © 2020 Joseph LaFreniere <joseph@lafreniere.xyz>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2023 Felix Lechner <felix.lechner@lease-up.com>
@@ -38,6 +39,29 @@ (define-module (gnu packages golang-check)
 ;;;
 ;;; Code:
 
+(define-public go-github-com-golangplus-testing
+  (package
+    (name "go-github-com-golangplus-testing")
+    (version "1.0.0")
+    (home-page "https://github.com/golangplus/testing")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url home-page)
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1a29m4zplf9m14k74lrb55dids2l17vx28sv0g3y3qcv1xygksiv"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/golangplus/testing"))
+    (propagated-inputs
+     (list go-github-com-golangplus-fmt))
+    (synopsis "Additions to Go's standard testing package")
+    (description "This package provides additions to Go's stdlib testing.")
+    (license license:bsd-3)))
+
 (define-public go-github-com-jacobsa-oglematchers
   (let ((commit "141901ea67cd4769c6800aa7bfdfc558fa22bda5")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index afc1a9bd30..cfd7fb73ad 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3006,28 +3006,6 @@ (define-public go-github-com-daviddengcn-go-colortext
     ;; dual-licensed
     (license (list license:bsd-3 license:expat))))
 
-(define-public go-github-com-golangplus-testing
-  (package
-    (name "go-github-com-golangplus-testing")
-    (version "1.0.0")
-    (home-page "https://github.com/golangplus/testing")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url home-page)
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32 "1a29m4zplf9m14k74lrb55dids2l17vx28sv0g3y3qcv1xygksiv"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/golangplus/testing"))
-    (propagated-inputs
-     (list go-github-com-golangplus-fmt))
-    (synopsis "Additions to Go's standard testing package")
-    (description "This package provides additions to Go's stdlib testing.")
-    (license license:bsd-3)))
-
 (define-public go-github-com-leodido-go-urn
   (package
     (name "go-github-com-leodido-go-urn")
-- 
2.41.0

